docs/overview/office-add-in-availability.md

```diff
@@ -985,7 +985,7 @@ To work as expected, your Office Add-in might depend on a specific Office host,
 - [Common API requirement sets](../reference/requirement-sets/office-add-in-requirement-sets.md)
 - [Add-in Commands requirement sets](../reference/requirement-sets/add-in-commands-requirement-sets.md)
 - [API Reference documentation](../reference/javascript-api-for-office.md)
-- [Update history for Office 365 ProPlus](/officeupdates/update-history-office365-proplus-by-date)
+- [Update history for Microsoft 365 Apps for enterprise](/officeupdates/update-history-office365-proplus-by-date)
```

Comment:
The title for this page is "Update history for Microsoft 365 Apps (listed by date)". The description is "These version and build numbers apply to Microsoft 365 Apps for enterprise, Microsoft 365 Apps for business, and the subscription versions of the desktop apps for Project and Visio." Should we restrict our reference to "enterprise"?

Reply:
Removed "for enterprise" so matches target topic.